cves/kernel/CVE-2016-6187.yml

```diff
@@ -1,479 +1,497 @@
-    resolve uncertainty.
+        A discussion can include debates, disputes, or polite talk about how to
+        resolve uncertainty.
 
-    Example include:
-      * Is this out of our scope?
-      * Is this a security?
-      * How should we fix this?
+        Example include:
+          * Is this out of our scope?
+          * Is this a security?
+          * How should we fix this?
 
-    Just because you see multiple comments doesn't mean it's a discussion.
-    For example:
-      * "Fix line 10". "Ok" is not what we call a discussion
-      * "Ping" (reminding people)
+        Just because you see multiple comments doesn't mean it's a discussion.
+        For example:
+          * "Fix line 10". "Ok" is not what we call a discussion
+          * "Ping" (reminding people)
 
-    Check the bugs reports, pull requests, and mailing lists archives.
+        Check the bugs reports, pull requests, and mailing lists archives.
 
-    These answers should be boolean.
-      discussed_as_security: true or false
-      any_discussion: true or false
+        These answers should be boolean.
+          discussed_as_security: true or false
+          any_discussion: true or false
 
-    Put any links to disagreements you found in the notes section, or any other
-    comment you want to make.
-  discussed_as_security:
-  any_discussion:
-  note:
+        Put any links to disagreements you found in the notes section, or any other
+        comment you want to make.
+    discussed_as_security: False
+    any_discussion: False
+    note: "No real notable discussion found"
 vouch:
-  question: |
-    Was there any part of the fix that involved one person vouching for
-    another's work?
-
-    This can include:
-      * signing off on a commit message
-      * mentioning a discussion with a colleague checking the work
-      * upvoting a solution on a pull request
-
-    Answer must be true or false.
-    Write a note about how you came to the conclusions you did, regardless of what your answer was.
-  answer:
-  note:
+    question: |
+        Was there any part of the fix that involved one person vouching for
+        another's work?
+
+        This can include:
+          * signing off on a commit message
+          * mentioning a discussion with a colleague checking the work
+          * upvoting a solution on a pull request
+
+        Answer must be true or false.
+        Write a note about how you came to the conclusions you did, regardless of what your answer was.
+    answer: True
+    note: |
+        "Patch was based on a patch from another developer, Vegard Nossum. It
+        was reported by that same person, reviewed by Tyler Hicks, and signed
+        off by John Johansen and James Morris. CC'd on the commit are also Al
+        Viro, Paul Moore, Stephen Smalley, Eric Paris, and Casey Schaufler. I
+        believe it was John Johansen that made the CVE request for this
+        vulnerability."
```

Comment:
You may not have the answer to this but is it suggested that Al Viro, Paul Moore or any of the other people mentioned actually contributed to the fix or maybe even the cause? So we can safely assume that they've looked at it and at the very least agree but I wonder if there's any indication that they were involved before it was ready to become a patch?